src/Manager/Environment/Name.tsx

```diff
@@ -6,15 +6,39 @@
 
 import React, { FC } from 'react';
 import Row from 'react-bootstrap/Row';
+import { execSync } from 'child_process';
+import path from 'path';
 
+import { checkExecArchitecture } from '../../helpers';
 import { Environment } from '../../state';
 
 import './style.scss';
 
-const Name: FC<{ environment: Environment }> = ({ environment }) => (
-    <Row noGutters className="toolchain-item-info h4 mb-0 pt-3">
-        nRF Connect SDK {environment.version}
-    </Row>
-);
+const isAppleSilicon =
+    process.platform === 'darwin' && execSync('uname -m').includes('arm64');
```

Comment:
This (although the implementation slightly differs), already exists in vscode.ts line 63. 
I'd imagine we should rather pop this into the helpers or import from vscode.ts.